.zuul.yaml

```diff
@@ -7,3 +7,33 @@
         - openstack-meta-content-provider:
             vars:
               cifmw_operator_build_meta_build: false
+        - watcher-operator-validation
+
+- job:
+    name: watcher-operator-base
```

Comment:
if it should never run itself then you could also add abstract but not worth blocking for